package.nls.json

```diff
@@ -96,7 +96,7 @@
     "jupyter.command.jupyter.gotoPrevCellInFile.title": "Go to Previous Cell",
     "jupyter.command.jupyter.showhistorypane.title": "Show Interactive Window",
     "jupyter.command.jupyter.createnewinteractive.title": "Create Interactive Window",
-    "jupyter.command.jupyter.selectjupyteruri.title": "Specify Jupyter server for connections",
+    "jupyter.command.jupyter.selectjupyteruri.title": "Specify Jupyter Server for Connections",
     "DataScience.jupyterNativeNotebookUriStatusLabelForLocal": "Jupyter Server: local",
     "DataScience.jupyterNativeNotebookUriStatusLabelForRemote": "Jupyter Server: Remote",
     "jupyter.command.jupyter.selectjupytercommandline.title": "Specify Jupyter command line arguments",
```

Comment:
Could you fix all commands? Not just 'selectjupyteruri'?

Reply:
can do!

Reply:
Changed all that I could identify as a command 👍